irispy/new_sji.py

```diff
@@ -274,7 +274,7 @@ def __repr__(self):
 
     def __getitem__(self, item):
         return utils.sequence.slice_sequence(self, item)
```

Comment:
I use `utils.sequence.slice_sequence(self, item)` rather than `index_as_cube` because the last function is calling a lot of function that are not relevant (creating as many error as functions) and return the same result. What do you think about it ?

Reply:
I think you should call ```index_as_cube```.  The extra checks it does make it more rigorous.  We just need to get it working here.  And I think simply calling ```utils.sequence.slice_sequence``` will not always be correct.  The way you should call ```index_as_cube``` it is:
```python
return self.index_as_cube[item]
```
Have you tried this?  If so, what errors are you getting?

Reply:
Now it works !